src/Umbraco.Cms.Integrations.Automation.Zapier/Components/NewContentPublishedComponent.cs

```diff
@@ -49,22 +49,23 @@ private void ContentServiceOnPublished(IContentService sender, ContentPublishedE
         {
             foreach (var node in e.PublishedEntities)
             {
-                var zapContentConfig = _zapConfigService.GetByName(node.ContentType.Name);
-                if (zapContentConfig == null || !zapContentConfig.IsEnabled) continue;
-
-                var content = new Dictionary<string, string>
+                if (_zapierSubscriptionHookService.TryGetByAlias(node.ContentType.Alias, out var zapContentConfig))
                 {
-                    { Constants.Content.Id, node.Id.ToString() },
-                    { Constants.Content.Name, node.Name },
-                    { Constants.Content.PublishDate, DateTime.UtcNow.ToString() }
-                };
+                    var content = new Dictionary<string, string>
+                    {
+                        {Constants.Content.Id, node.Id.ToString()},
+                        {Constants.Content.Name, node.Name},
+                        {Constants.Content.PublishDate, DateTime.UtcNow.ToString()}
+                    };
 
-                var t = Task.Run(async () => await _zapierService.TriggerAsync(zapContentConfig.WebHookUrl, content));
+                    var t = Task.Run(
+                        async () => await _zapierService.TriggerAsync(zapContentConfig.HookUrl, content));
 
-                var result = t.Result;
+                    var result = t.Result;
```

Comment:
You could move these lines into a helper class - either a static method, or another class that you inject an instance of - so you can share the code between the V8 and V9 versions.